pkg/remote/aws/api_gateway_account_enumerator.go

```diff
@@ -0,0 +1,51 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+type ApiGatewayAccountEnumerator struct {
+	repository repository.ApiGatewayRepository
+	factory    resource.ResourceFactory
+}
+
+func NewApiGatewayAccountEnumerator(repo repository.ApiGatewayRepository, factory resource.ResourceFactory) *ApiGatewayAccountEnumerator {
+	return &ApiGatewayAccountEnumerator{
+		repository: repo,
+		factory:    factory,
+	}
+}
+
+func (e *ApiGatewayAccountEnumerator) SupportedType() resource.ResourceType {
+	return aws.AwsApiGatewayAccountResourceType
+}
+
+func (e *ApiGatewayAccountEnumerator) Enumerate() ([]*resource.Resource, error) {
+	account, err := e.repository.GetAccount()
+	if err != nil {
+		return nil, remoteerror.NewResourceListingError(err, string(e.SupportedType()))
+	}
+
+	results := make([]*resource.Resource, 1)
+
+	if account != nil {
+		attrs := map[string]interface{}{}
+		if account.CloudwatchRoleArn != nil && *account.CloudwatchRoleArn != "" {
+			attrs["cloudwatch_role_arn"] = *account.CloudwatchRoleArn
+		}
+
+		results = append(
+			results,
+			e.factory.CreateAbstractResource(
+				string(e.SupportedType()),
+				"api-gateway-account",
```

Comment:
Is this hardcoded on terraform provider side too ?

Reply:
Yes exactly it is hardcoded in the provider 
![image](https://user-images.githubusercontent.com/8110579/134942356-edee51d9-d470-4df8-b733-f179bb386eb2.png)

Reply:
🆗 👌🏻